### ISSUE TITLE
Hi, there are some issues about the infoNCEloss

### ISSUE THREAD

**Issue by LanXiaoPang613** [lxysl/PLReMix#1]
Hi, there are some issues about the infoNCEloss:

when we define the info_nce_loss in the code as follows: 
**info_nce_loss = InfoNCELoss(temperature=0.1,
                                batch_size=args.batch_size * 2,
                                flat=('flat' in args.crl),
                                n_views=8 if args.mcrop else 2)**
the labels size in the function of infoNCELoss is batch_size x 2 x n_views, while the size of the features are batch_size x n_views. Since the sizes of these two params are not matched, the calculation of the infoNCEloss is not correct.
![image](https://github.com/lxysl/PLReMix/assets/35187545/0f90cd89-8bac-4020-a1d1-f16b1874ee44)

**Comment by lxysl** [comment]
The `features` are comprised of two parts: `labeled` and `unlabeled`, each with a size equal to `batch_size*n_views`.

**Comment by LanXiaoPang613** [comment]
> The `features` are comprised of two parts: `labeled` and `unlabeled`, each with a size equal to `batch_size*n_views`.

I know, but when this function is used to perform warm-up training, it will be incorrect. If possible, you can check it.

**Comment by lxysl** [comment]
If you are asking the warm-up stage, I guess this line is what you are looking for: https://github.com/lxysl/PLReMix/blob/77cf96bd964b076d52a58501bc8d19d5d853307f/data/data_loader.py#L361

**Comment by LanXiaoPang613** [comment]
> If you are asking the warm-up stage, I guess this line is what you are looking for:
> 
> https://github.com/lxysl/PLReMix/blob/77cf96bd964b076d52a58501bc8d19d5d853307f/data/data_loader.py#L361

OKay，i ignore this line. Thank you.

**Comment by LanXiaoPang613** [comment]
If possible, could you hare me a training log about the PLRemix on webvision, it seems i met some problems when training on this set.
